src/adaptors/dforce/index.js

```diff
@@ -0,0 +1,44 @@
+const utils = require('../utils');
+
+const getApiUrl = (chain) =>
+  `https://app.dforce.network/general/markets?network=${chain}`;
+
+const NETWORKS = {
+  ethereum: 'mainnet',
+  binance: 'bsc',
+  polygon: 'Polygon',
+  arbitrum: 'ArbitrumOne',
+  optimism: 'Optimism',
+};
+
+const getApy = async () => {
+  const markets = await Promise.all(
+    Object.keys(NETWORKS).map(async (network) => {
+      return {
+        network,
+        data: await utils.getData(getApiUrl(NETWORKS[network])),
+      };
+    })
+  );
+
+  const pools = markets
+    .map(({ network, data: { supplyMarkets } }) => {
+      return supplyMarkets.map((market) => ({
+        pool: market.address,
```

Comment:
seems like this pool `0xb3ab7148cCCAf66686AD6C1bE24D83e58E6a504e` is repeated, occurs twice (on Abritrum and Polygon). We could add the chain id as suffix